crates/sui-sdk/examples/read_api.rs

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+mod utils;
+
+use sui_sdk::rpc_types::{SuiGetPastObjectRequest, SuiObjectDataOptions};
+use sui_sdk::types::base_types::ObjectID;
+use sui_sdk::SuiClientBuilder;
+use utils::sui_address_for_examples;
+
+#[tokio::main]
+async fn main() -> Result<(), anyhow::Error> {
+    let sui = SuiClientBuilder::default().build_testnet().await?; // testnet Sui network
+    println!("Sui testnet version{:?}", sui.api_version());
+    let active_address = sui_address_for_examples().await?; // get a random address for examples. Check utils module if you want to use a local wallet
+
+    // ************ READ API ************ //
+
+    println!("// ************ READ API ************ //\n");
+    // Owned Objects
```

Comment:
thoughts on an example for reading transactions too?

Reply:
Good point. Any pointers? I'll try to "imagine" something and come up with an example.